README.md

```diff
@@ -3,9 +3,11 @@
 
 [![Build Status](https://travis-ci.org/jupyter/repo2docker.svg?branch=master)](https://travis-ci.org/jupyter/repo2docker)
 [![Documentation Status](https://readthedocs.org/projects/repo2docker/badge/?version=latest)](http://repo2docker.readthedocs.io/en/latest/?badge=latest)
- 
-**jupyter-repo2docker** is a tool to build, run and push docker images from source code repositories.
 
+**jupyter-repo2docker** is a tool to build, run and push Docker images from source code repositories.
```

Comment:
Can we tweak this slightly:

**jupyter-repo2docker**, a convenient tool, takes a source, such as a GitHub repository, as input to build, run and push Docker images.